Lessons/Lesson-19-Css/html/index.html

```diff
@@ -73,23 +81,58 @@ <h3>Register</h3>
         body.className += " " + styleAttr;
     }
 
-    class MyGlobalScopeClass {
-        constructor() {
-          this.myProp = "prop";
-        }
+    function validate(evt) 
+    {
+        let inputNameResult =  ValidateInput('name');
+        if (!inputNameResult) return false;
 
-        getMyProp() {
-            return this.myProp;
-        }
+        let inputLastNameResult =  ValidateInput('last_name');
+        if (!inputLastNameResult) return false;
+
+        let checkboxEl = document.getElementById('accept');
+        if (!checkboxEl.checked)
+         {
+            alert("you must accept terms and conditions!");
+            return false;
+         }
 
-        setMyProp(val) {
-            this.myProp = val;
-        }
-    };
 
-    function validate(evt) {
-        return confirm("are you sure?");
+        return true;
+        //return confirm("are you sure?");
+    }
+
+    function ValidateInput(id)
+    {
+        let inputEl = document.getElementById(id);
+        if (inputEl == undefined) return true;
+        let inputValue = inputEl.value;
+        if (inputValue.trim() == "") 
+        {
+            alert(id + " cannot be empty!");
+            return false;
+        }
+        let arrNumber = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+        let arrForbiddenSymbols = ["+", "<", ">", "^", "@", "\"", "\'", ",", ";", ":", "[", "]", "}", "{"];
+        //check  1-st symbol
+        let nameFirstSymbol = inputValue[0];
+        if (arrNumber.indexOf(Number(nameFirstSymbol)) !== -1)
+        {
+            alert(id + ": the first character must not be a number!");
+            return false;
+        }
+        //check forbidden symbols
+        for (let index = 0; index < inputValue.length; index++) 
```

Comment:
for this purpose it makes sense to use RegExp, should not affect performance and a good opportunity to start learning it